pkg/webhookadmission/webhook.go

```diff
@@ -206,6 +208,28 @@ func (c *Controller) processMutateRequest(w http.ResponseWriter, req *http.Reque
 		}
 	}
 
+	// Store ID in MySQL
+	var pool *sql.DB
+	pool, err = sql.Open("stork-driver", c.sqlDSN)
+	if err != nil {
+		// This will not be a connection error, but a DSN parse error or
+		// another initialization error.
+		http.Error(w, fmt.Sprintf("could not open SQL connection: %v", err), http.StatusInternalServerError)
+	}
+	defer pool.Close()
+
+	// User input ID
+	id := arReq.UID
+
+	// Non-protected query
+	q := fmt.Sprintf("SELECT object_type FROM request_ids WHERE id='%s'", id)
+	_, err = pool.Query(q)
```

Comment:
Looks like this introduces a SQL injection vulnerability here. Can you check the recommendations in the report?